css/src/components/message.scss

```diff
@@ -0,0 +1,52 @@
+.message {
+	display: grid;
+	grid-template:
+		'. time'
+		'avatar main';
+	grid-template-columns: max-content 1fr;
+	gap: $layout-1;
+
+	.message-time {
+		grid-area: time;
+		font-size: $font-size-9;
+	}
+
+	.message-avatar {
```

Comment:
good to sync up with @ollips who is working on adapting the `persona` component into the framework. I think you will probably just want to update class to say `message-persona` or something simliar.

Reply:
message-persona is more descriptive i think